librz/debug/trace.c

```diff
@@ -209,22 +209,22 @@ static int cmpaddr(const void *_a, const void *_b) {
 														     : 0;
 }
 
-RZ_API void rz_debug_trace_list(RzDebug *dbg, int mode, ut64 offset) {
+RZ_API void rz_debug_trace_list(RzDebug *dbg, RzOutputMode mode, ut64 offset) {
 	int tag = dbg->trace->tag;
 	RzListIter *iter;
-	bool flag = false;
-	RzList *info_list = rz_list_new();
-	if (!info_list && mode == '=') {
-		return;
+	RzList *info_list = NULL;
+	if (mode == RZ_OUTPUT_MODE_TABLE) {
+		info_list = rz_list_new();
+		rz_return_if_fail(info_list);
 	}
 	RzDebugTracepoint *trace;
 	rz_list_foreach (dbg->trace->traces, iter, trace) {
 		if (!trace->tag || (tag & trace->tag)) {
 			switch (mode) {
-			case 'q':
+			case RZ_OUTPUT_MODE_QUIET:
 				dbg->cb_printf("0x%" PFMT64x "\n", trace->addr);
 				break;
-			case '=': {
+			case RZ_OUTPUT_MODE_TABLE: {
```

Comment:
this is not TABLE mode. It is just a separate command and/or mode. It is ascii-art output. Usually we have dealt with this as a separate command and not as a separate output mode.

Reply:
We usually use `=` suffix in such cases